set-quota-auditor.sh

```diff
@@ -0,0 +1,22 @@
+# This script walks through each org in a Cloud Foundry instance setting
+# OrgAuditor and SpaceAuditor permissions at each level.
+#
+# Usage:
+# 
+#  set-quota-auditor.sh AUDIT_USER
+USER_EMAIL=$1
+
+# Strip the host portion of the email supplied.
+USER_NAME="${USER_EMAIL%%@*}"
+
+cf orgs \
+  | awk 'm;/^name/{m=1}' \
+  | while read ORG_NAME
+      do cf set-org-role $USER_NAME $ORG_NAME OrgAuditor
+      cf target -o $ORG_NAME
+      cf spaces \
+      | awk 'm;/^name/{m=1}' \
+      | while read SPACE_NAME
+          do cf set-space-role $USER_NAME $ORG_NAME $SPACE_NAME SpaceAuditor
```

Comment:
Yes, it seems to be necessary.

With `OrgAuditor` alone @ramirezg was getting back empty responses querying spaces for services, which are space-bound. 

I imagine it could be another case of the situation we had/have with `OrgManager` and `SpaceManager` where the backend knows how to enumerate permissions given the right API calls. 

We'd have to check which calls @ramirezg is making to see if there's are better / preferred endpoint(s) to the same end.